packages/core/content-manager/server/controllers/collection-types.js

```diff
@@ -432,7 +432,36 @@ module.exports = {
       return ctx.forbidden();
     }
 
-    const number = await entityManager.getNumberOfDraftRelations(id, model);
+    const number = await entityManager.countDraftRelations(id, model);
+
+    return {
+      data: number,
+    };
+  },
+  async countManyEntriesDraftRelations(ctx) {
+    const { userAbility } = ctx.state;
+    const ids = ctx.request.query.ids;
+    const { model } = ctx.params;
+
+    const entityManager = getService('entity-manager');
+    const permissionChecker = getService('permission-checker').create({ userAbility, model });
+
+    if (permissionChecker.cannot.read()) {
+      return ctx.forbidden();
+    }
+
+    const populate = await getService('populate-builder')(model)
+      .populateDeep()
+      .countRelations()
+      .build();
+
+    const entities = await entityManager.find(ids, model, { populate });
+
+    if (!entities) {
+      return ctx.notFound();
+    }
+
+    const number = await entityManager.countManyEntriesDraftRelations(ids, model);
```

Comment:
was wrong when mentioned this should be populated :) You are actually using `getDeepPopulateDraftCount` inside the service!

I think this should be enough
```suggestion
    const entities = await entityManager.find(ids, model);

    if (!entities) {
      return ctx.notFound();
    }

    const number = await entityManager.countManyEntriesDraftRelations(ids, model);
```

Reply:
ah ok, thanks for the suggestion, done...Do you think the `!entities` check is enough or do we need to control also every single entity?

Reply:
could you elaborate on "do we need to control also every single entity?" ?

In what sense? permission wise?

If it's about that, I think this is enough, as we are not showing any data but the number of entries with draft relations.